Source/Core/CameraLimiter.js

```diff
@@ -0,0 +1,70 @@
+define([
+    '../Core/Cartesian3',
+    '../Core/Check',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/HeadingPitchRoll',
+    '../Core/Math'
+], function(
+    Cartesian3,
+    Check,
+    defaultValue,
+    defined,
+    HeadingPitchRoll,
+    CesiumMath) {
+    'use strict';
+
+    /**
+     * The CameraLimiter is used to limit a {@link Camera}'s position (using a bounding object).
+     *
+     * If set, the camera of this limiter will be constrained by location.
+     * The camera ignores position limits in {@link SceneMode#SCENE2D} view and {@link SceneMode#COLUMBUS_VIEW}.
+     *
+     * @alias CameraLimiter
+     * @constructor
+     *
+     * @param {AxisAlignedBoundingBox|BoundingSphere|OrientedBoundingBox} [options.boundingObject] Assigned to the boundingObject attribute which can be used to specify limits on locations.
+     */
+    function CameraLimiter(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        /**
+         * The BoundingObject. Used to restrict camera positions of the {@link Camera} holding this CameraLimiter.
```

Comment:
Don't just repeat the name of the variable, that's not very descriptive.

"The volume used to restrict a {@link Camera}'s position." ?